libmamba/include/mamba/core/subdirdata.hpp

```diff
@@ -44,20 +44,6 @@ namespace mamba
     class MSubdirData
     {
     public:
-        /**
-         * Constructor.
-         * @param name Name of the subdirectory (<channel>/<subdir>)
-         * @param repodata_url URL of the repodata file
-         * @param repodata_fn Local path of the repodata file
-         * @param possible packages caches
-         * @param is_noarch Local path of the repodata file
-         */
-        MSubdirData(const std::string& name,
-                    const std::string& repodata_url,
```

Comment:
I think we may want to keep one where we can give explicitly the repodata url (that might also be used by the conda integration).

Reply:
I've added an argument to specify the repodata file name (and there is already an argument to pass the "base" URL)